### DISCUSSION TITLE
nbb in the browser?

### DISCUSSION THREAD

**Discussion by icosahedron** [babashka/nbb#110] [General]
Is it possible to run `nbb` in the browser?  Given a string of clojure, would it interpret it?

**Comment by borkdude** [comment]
@icosahedron Yes, but it's called scittle: https://babashka.org/scittle/

Depending on your needs, you can also use SCI directly: https://github.com/borkdude/sci

**Reply by icosahedron** [reply]
Thanks! Will definitely look at that.
